docs/cli-reference/dfx-start.md

```diff
@@ -57,3 +57,32 @@ You can view the current process identifier (`pid`) for the local canister execu
 ``` bash
 more .dfx/pid
 ```
+
+## Local Server Configuration
+
+### The Shared Local Network
+
+By default, `dfx start` manages a single replica that is independent of any given local project.  Running `dfx deploy` and other commands will manage canisters on this single local network, in the same way that `dfx deploy --network ic` deploys separate projects to mainnet.
+
+If run from outside any dfx project, or if dfx.json does not define the `local` network, then `dfx start` looks for the `local` network definition in `$HOME/.config/dfx/networks.json`.  If this file does not exist or does not contain a definition for the `local` network, then dfx uses the following default definition:
+
+```
+{
+  "local": {
+    "bind": "127.0.0.1:8000",
+    "type": "ephemeral"
+  }
+}
+```
+
+dfx stores data for the shared local network in one of the following locations, depending on your operating system:
+- `$HOME/.local/share/dfx/network/local` (Linux)
+- `$HOME/Library/Application Support/org.dfinity.dfx/network/local` (Macos)
+
+### Project-Specific Local Networks
+
+If dfx.json defines the `local` network, then `dfx start` will use this definition and store network data files under `\<project dir\>/.dfx/network/local`. 
+
+Such project-specific networks are deprecated, slated to be removed after February 2023.  We encourage you to remove any definitions of the `local` network from your project's dfx.json file and instead use the default shared local network.
```

Comment:
```suggestion
Such project-specific networks are deprecated, slanted to be removed after February 2023.  We encourage you to remove any definitions of the `local` network from your project's dfx.json file and instead use the default shared local network.
```

Reply:
"slanted" wouldn't make sense here.  "slated" means planned.  I'll change this to "planned."

https://dictionary.cambridge.org/us/dictionary/english/slated

> be slated
 
> to be [expected](https://dictionary.cambridge.org/us/dictionary/english/expected) to [happen](https://dictionary.cambridge.org/us/dictionary/english/happen) in the [future](https://dictionary.cambridge.org/us/dictionary/english/future) or to be [expected](https://dictionary.cambridge.org/us/dictionary/english/expected) to be or do something in the [future](https://dictionary.cambridge.org/us/dictionary/english/future)